frontend/src/metabase/query_builder/components/notebook/NotebookStep.jsx

```diff
@@ -254,9 +255,10 @@ const ActionButton = ({
       iconVertical={large}
       iconSize={large ? 18 : 14}
       onClick={onClick}
+      aria-label={label}
```

Comment:
So, `findByRole("option", { name: "xxx" })` would work, since there's also an icon in front of the next which makes its way into the accessibility name.